pkgs/development/rocm-modules/6/hipblas/default.nix

```diff
@@ -63,14 +73,15 @@ stdenv.mkDerivation (finalAttrs: {
     '')
   ];
 
+  propagatedBuildInputs = [ hipblas-common ];
```

Comment:
Not a huge deal, but shouldn't header-only libraries such as `hipblas-common` and `rocprim` be in `(propagated)NativeBuildInputs` instead of `buildInputs`?

Reply:
IDK. Asked in matrix and @K900 says buildInputs for header-only libs